scripts/src/main/resources/scripts/command/docker

```diff
@@ -150,6 +171,34 @@ function doIsRancherInstalled() {
     fi
 }
 
+
+function doIsVPNKitInstalled() {
```

Comment:
Just for the record: In `pascalCase` and `CamlCase` never use CAPITALIZED abbreviations like `VPN`.
For Java coding we had documented this as explicit convention here:
https://github.com/devonfw/devon4j/blob/master/documentation/coding-conventions.asciidoc#naming

For bash we did not write explicit coding conventions but it makes sense to follow this same rule(s) for all languages.

This is just a nice-to-have improvement you should consider for your next update. It will not block this PR from being approved and merged.